pkg/controller/controller.go

```diff
@@ -864,6 +865,8 @@ func (p *csiProvisioner) Delete(volume *v1.PersistentVolume) error {
 				klog.Errorf("Failed to get credentials for volume %s: %s", volume.Name, err.Error())
 			}
 			req.Secrets = credentials
+		} else {
+			klog.Errorf("failed to get storageclass: %s", storageClassName)
```

Comment:
This should be a warning with a message like "failed to get storageclass, proceeding to delete without secrets"

Reply:
Done.